lua/autorun/sh_jackal_convars.lua

```diff
@@ -1,8 +1,10 @@
 -- replicated convars have to be created on both client and server
 CreateConVar("ttt_jackal_armor_value", 30, {FCVAR_ARCHIVE, FCVAR_NOTIFY, FCVAR_REPLICATED})
+CreateConVar("ttt_jackal_spawn_siki_deeagle", 1, {FCVAR_ARCHIVE, FCVAR_NOTIFY, FCVAR_REPLICATED})
```

Comment:
`ttt_jackal_spawn_siki_deeagle` is wrong. Better use `ttt_jackal_spawn_siki_deagle`

Reply:
I totally agree deagle doesn't have three e in the name